src/lower.cc

```diff
@@ -220,6 +220,7 @@ Halide::Pipeline lower(Builder builder, std::vector<Node>& nodes, bool implicit_
     // Constructing Generator object and setting static parameters
     std::unordered_map<NodeID, Halide::Internal::AbstractGeneratorPtr, NodeID::StringIDHash> bbs;
     for (auto n : nodes) {
+        n.validate_port_address_alignment();
```

Comment:
Let's separate loops to keep clear code structure.

Reply:
```
for (auto n : nodes) {
        n.validate_port_address_alignment();
}
```